cmake/modules/FindNLopt.cmake

```diff
@@ -28,7 +28,7 @@ if(NOT NLopt_DIR)
 
 	set(NLopt_FOUND TRUE)
 
-	set(_NLopt_LIB_NAMES "nlopt")
+	set(_NLopt_LIB_NAMES "nlopt" "nloptd")
```

Comment:
is there a reason to link the debug version lib here?

Reply:
To allow for proper debug building. In hindsight, it could be set to only include that lib during debug builds

Reply:
Yeah, I feel it's wrong to include debug libs in release build or vise-versa